tdei-api.json

```diff
@@ -1706,7 +1706,7 @@
         },
         "responses": {
             "Unauthorized": {
-                "description": "This request is unauthorized. appID is invalid. Please obtain a valid application ID (appID)."
+                "description": "This request is unauthorized. The AppID or Authorization Token provided is not authorized to perform this request."
```

Comment:
Should we change this to "This request is unauthorized. The API_KEY or the authorization token provided is invalid to perform this request"?